subsys/bluetooth/controller/Kconfig

```diff
@@ -344,6 +344,13 @@ config BT_CTLR_SDC_PERIODIC_ADV_RSP_RX_BUFFER_COUNT
 	  until the host is able to pull the periodic advertising response reports.
 
 	  A value of 0 will disable reception of subevent responses.
+
+config BT_CTLR_SDC_PERIODIC_ADV_RSP_FAILURE_REPORTING_ENABLE
+	int "Failure reporting for the PAwR advertiser"
```

Comment:
As the failures will only be a generated as response reports, I suggest we change it to:
"Failure response reports for PAwR Advertiser"
To make it more explicit and easier to understand what sort of failure reporting that can be enabled.